### ISSUE TITLE
Dart generator test failure: Colour

### ISSUE THREAD

**Issue by rachel-fenichel** [google/blockly#1202]
Error message: 
```
.FF................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................

FAIL: blend
Expected: #ff2900
Actual: #ff6600

FAIL: from rgb
Expected: #ff6600
Actual: #ffff00

Ran 803 tests.

FAILED (failures=2)
```

**Comment by TonyLianLong** [comment]
The reason why these tests fail is that the `color_rgb` function behaves differently in Dart than in other languages. In Dart, this function takes parameters ranging from 0 to 1.0 while in other languages such as Lua the counterpart function takes parameters ranging from 0 to 100.
Now I have already solved the problem in my code and ready for creating a pull request.

**Comment by AnmAtAnm** [comment]
Fixed in #1210.  Thanks, Tony!
